### PR TITLE
CNV-13359: VMI to VM

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.13+

Issue: [CNV-13359](https://issues.redhat.com//browse/CNV-13359)

Link to docs preview: https://56165--docspreview.netlify.app/openshift-enterprise/latest/virt/logging_events_monitoring/virt-monitoring-vm-health.html

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/virt-about-readiness-liveness-probes.adoc [hunk 1]
```diff
@@ -7,16 +7,16 @@
...
+Use readiness and liveness probes to detect and handle unhealthy virtual machines (VMs). You can include one or more probes in the specification of the VM to ensure that traffic does not reach a VM that is not ready for it and that a new VM is created when a VM becomes unresponsive.
 
-A _readiness probe_ determines whether a VMI is ready to accept service requests. If the probe fails, the VMI is removed from the list of available endpoints until the VMI is ready.
+A _readiness probe_ determines whether a VM is ready to accept service requests. If the probe fails, the VM is removed from the list of available endpoints until the VM is ready.
 
-A _liveness probe_ determines whether a VMI is responsive. If the probe fails, the VMI is deleted and a new instance is created to restore responsiveness.
+A _liveness probe_ determines whether a VM is responsive. If the probe fails, the VM is deleted and a new VMs is created to restore responsiveness.
```
Comment (stu-gott) [inline modules/virt-about-readiness-liveness-probes.adoc:14]: NIT: "a new VM is created"...

**Review by stu-gott — COMMENTED** [review]
looks good! one tiny nit.

File: virt/logging_events_monitoring/virt-monitoring-vm-health.adoc [hunk 2]
```diff
@@ -18,8 +18,6 @@ include::modules/virt-define-http-liveness-probe.adoc[leveloffset=+1]
 
 include::modules/virt-define-guest-agent-ping-probe.adoc[leveloffset=+1]
 
-include::modules/virt-template-vm-probe-config.adoc[leveloffset=+1]
-
```
Comment (GroceryBoyJr) [inline virt/logging_events_monitoring/virt-monitoring-vm-health.adoc:22]: Question: did you mean to delete this module?

**Review by GroceryBoyJr — COMMENTED** [review]
@sabrinajess only one small question. Otherwise LGTM!

File: virt/logging_events_monitoring/virt-monitoring-vm-health.adoc (hunk 2 above)
Comment (sabrinajess) [reply virt/logging_events_monitoring/virt-monitoring-vm-health.adoc:22]: I did! Mentioned in the JIRA but not here. Thanks!

**Comment by kbidarkar** [comment]
LGTM

**Pull request merged**